### PR TITLE
CN-3352 CN-3353 CN-3355 prepayment screens

### PR DESCRIPTION
**Pull request by srkela**
PR Checklist:
- [x] I have named the PR correctly
- [x] I have added at least two reviewers
- [x] I have assigned the PR to myself
- [x] I have merged develop into my feature branch and resolved conflicts
- [x] I have tested all new features locally
- [x] I have tested relevant flow locally

### How to name a PR
- If it includes only one issue [CN-1234 Issue name]
- If it includes many issues [CN-1234, CN-2345, CN-3456 What was updated/changed/added]
- If the PR is not ready yet, add [DRAFT] in front of the name

### How to test locally
1. Test happy path
2. Test other paths
3. Remove all warnings

### How to relevant flow locally
- If working on part of CelPay, check the whole happy path

### CODE REVIEW

File: app/components/organisms/PaymentCards/PaymentCards.styles.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// import STYLES from '../../../constants/STYLES';
```
Comment (CrnogoracIvan) [inline app/components/organisms/PaymentCards/PaymentCards.styles.js]: Delete commented line.

File: app/components/screens/ChoosePrepaymentMethod/ChoosePrepaymentMethod.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+// import PropTypes from 'prop-types';
```
Comment (CrnogoracIvan) [inline app/components/screens/ChoosePrepaymentMethod/ChoosePrepaymentMethod.js]: Delete comment line.

File: app/components/screens/ChoosePrepaymentMethod/ChoosePrepaymentMethod.styles.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// import STYLES from '../../../constants/STYLES';
```
Comment (CrnogoracIvan) [inline app/components/screens/ChoosePrepaymentMethod/ChoosePrepaymentMethod.styles.js]: Delete comment line.

File: app/components/screens/LoanPrepaymentPeriod/LoanPrepaymentPeriod.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+// import PropTypes from 'prop-types';
```
Comment (CrnogoracIvan) [inline app/components/screens/LoanPrepaymentPeriod/LoanPrepaymentPeriod.js]: Delete comment line.

File: app/components/screens/LoanPrepaymentPeriod/LoanPrepaymentPeriod.styles.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+// import STYLES from '../../../constants/STYLES';
```
Comment (CrnogoracIvan) [inline app/components/screens/LoanPrepaymentPeriod/LoanPrepaymentPeriod.styles.js]: Delete comment line.

File: app/components/screens/PaymentCel/PaymentCel.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+// import { View } from 'react-native';
```
Comment (CrnogoracIvan) [inline app/components/screens/PaymentCel/PaymentCel.js]: I assume that this on this file work is in progress :)?

File: app/components/screens/PaymentCel/PaymentCel.styles.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
+// import STYLES from '../../../constants/STYLES';
```
Comment (CrnogoracIvan) [inline app/components/screens/PaymentCel/PaymentCel.styles.js]: Delete comment line.

File: app/components/screens/PersonalInformation/PersonalInformation.styles.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
+// import STYLES from '../../../constants/STYLES';
```
Comment (CrnogoracIvan) [inline app/components/screens/PersonalInformation/PersonalInformation.styles.js]: Delete comment line.

File: app/components/screens/PaymentCel/PaymentCel.js (hunk 6 above)
Comment (srkela) [reply app/components/screens/PaymentCel/PaymentCel.js]: This PR was in draft until now :)

File: app/components/organisms/PaymentCards/PaymentCards.js [hunk 9]
```diff
@@ -0,0 +1,218 @@
+import React, { Component } from 'react';
```
Comment (CrnogoracIvan) [inline app/components/organisms/PaymentCards/PaymentCards.js]: PaymentCards component break to single PaymentCard component, and then use it in ChoosePrepaymentMethod screen.

**Pull request merged**